app/controllers/lawyers_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LawyersController < ApplicationController
+  def index
+    @lawyer = Lawyer.all
+  end
+
+  def create
+    @current_user = current_user
+    @lawyer = current_user.Lawyer.new(lawyer_params)
+    @lawyer.save
+    redirect_to lawyers_path
+  end
```

Comment:
Our app is an API-based server, so we don't need to redirect to somewhere after creating a lawyer, in this case, we need to render JSON so our client app can consume it.
```suggestion
  def create
    @lawyer = current_user.Lawyer.new(lawyer_params)
    if @lawyer.save
      render json: { result: 'Lawyer created successfully' }
    else
      render json: { result: 'Sometinh went wrong' }, status: :unprocessable_entity
    end
  end
```